src/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/1097d30f85.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>To do-list</title>
+  </head>
+  <body>
+    <form class="todo-form">
+      <div class="title">
+        <p>Today's To Do</p>
```

Comment:
- [ ] [OPTIONAL] Kindly, use semantic tags in your project. You could use a header tag instead of a generic div. Using semantic tags gives you more control of your project.
- [ ] [OPTIONAL] For the title instead of p tag use h1